department-of-veterans-affairs/caseflow
# Pamatya/appeals 26561

Resolves : Move BusinessLine organization seed data [APPEALS-26561](https://jira.devops.va.gov/browse/APPEALS-26561)

# Description
1. made a new json file and folder to include Businessline organizations info. 
2. added a class under seeds module to import Business line organization. thi
3. added a method in sanitized json class to differentiate different folder path. 

## Acceptance Criteria
- [ ] Business Line Organization should be imported during the seeding process.
- [ ] when running make reset all the import should work as it was doing previously

## Testing Plan

1. Go to  [APPEALS-26561](https://jira.devops.va.gov/browse/APPEALS-26561)or list them below
2. Check if business line count is 9. 
3. check if all the business line has been imported properly. 
4. might need some regression testing. 


# Frontend
## User Facing Changes

 BEFORE|AFTER
 ---|---

## Storybook Story
*For Frontend (Presentation) Components*

# Backend
## Database Changes
*Only for Schema Changes*


## Integrations: Adding endpoints for external APIs

# Best practices
## Code Documentation Updates

## Tests
### Test Coverage
Did you include any test coverage for your code? Check below:

### Code Climate
Your code does not add any new code climate offenses? If so why?

## Monitoring, Logging, Auditing, Error, and Exception Handling Checklist

### Monitoring

### Logging

### Auditing

### Error Handling

### Exception Handling
